tests/org.jboss.tools.docker.ui.bot.test/pom.xml

```diff
@@ -11,12 +11,13 @@
 	<artifactId>org.jboss.tools.docker.ui.bot.test</artifactId>
 	<packaging>eclipse-test-plugin</packaging>
 	<properties>
-		<systemProperties>-DdockerServerURI=${dockerServerURI} -DunixSocket=${unixSocket} -DdockerHubUsername=${dockerHubUsername} -DdockerHubemail=${dockerHubEmail} -DdockerHubPassword=${dockerHubPassword}</systemProperties>
+		<systemProperties>-DdockerServerURI=${dockerServerURI} -DunixSocket=${unixSocket} -DdockerHubUsername=${dockerHubUsername} -DdockerHubemail=${dockerHubEmail} -DdockerHubPassword=${dockerHubPassword} -DdockerComposePath=${dockerComposePath}</systemProperties>
 		<unixSocket>unix:///var/run/docker.sock</unixSocket>
 		<dockerServerURI></dockerServerURI>
 		<dockerHubUsername></dockerHubUsername>
 		<dockerHubEmail></dockerHubEmail>
 		<dockerHubPassword></dockerHubPassword>
+		<dockerComposePath>/usr/local/bin</dockerComposePath>
```

Comment:
Yes, this is default installation folder on Linux. I also plan to support Mac and Win, but not now.

Thanks for comments:)